pod/perlvar.pod

```diff
@@ -476,10 +476,10 @@ sanity-checked.
 
 The C<$OLD_PERL_VERSION> form was added in Perl v5.20.0 for historical
 reasons but its use is discouraged. (If your reason to use C<$]> is to
-run code on old perls then referring to it as C<$OLD_PERL_VERSION> would
```

Comment:
This seems to definitely be referring to version of the interpreter, not the language.

Reply:
they may coincide, but usually the version is of the language

Reply:
I agree this instance is referring to the version of the Perl language.